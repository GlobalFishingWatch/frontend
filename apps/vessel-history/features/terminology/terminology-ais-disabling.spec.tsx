import { Trans, useTranslation } from 'react-i18next'
import { render } from '@testing-library/react'

import TerminologyAisDisabling from './terminology-ais-disabling'

jest.mock('react-i18next')

describe('TerminologyAisDisabling', () => {
  const mockUseTranslation: jest.Mock = useTranslation as jest.Mock
  const mockTrans: jest.Mock = Trans as jest.Mock
  beforeEach(() => {
    mockUseTranslation.mockReturnValue({
      t: (_ns: any, fallback: any) => fallback,
    })
    mockTrans.mockImplementation(({ i18nKey, children }) => children)
  })

  afterEach(() => {
    jest.clearAllMocks()
  })
  it('should render successfully', () => {
    const { baseElement } = render(<TerminologyAisDisabling />)
    expect(baseElement).toMatchSnapshot()
  })
})
