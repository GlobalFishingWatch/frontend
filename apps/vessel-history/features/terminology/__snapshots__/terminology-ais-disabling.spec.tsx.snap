// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TerminologyAisDisabling should render successfully 1`] = `
<body>
  <div>
    <p
      class="container"
    >
      A likely disabling event is indicated where a vessel may have intentionally turned off its AIS device, i.e. where a vessel has 'gone dark.' These events are gaps in AIS transmissions that our models predict are the result of intentional disabling rather than poor signal or similar factors.
    </p>
  </div>
</body>
`;
